packages/create-issue/package.json

```diff
@@ -41,10 +41,9 @@
     "test": "# jest",
     "watch": "tsc -p . --watch"
   },
-
-
-
   "devDependencies": {
+    "@octokit/graphql": "^4.8.0",
+    "@octokit/rest": "^18.12.0",
```

Comment:
These is dulicated. We need to remove these from devDependencies.

Reply:
Fixed :)